content/blog/2022/10/2022-10-10-plugin-health-scoring-system-report.adoc

```diff
@@ -0,0 +1,82 @@
+:tags:
+- gsoc
+- gsoc2022
+- healthscore
+- probes
+- plugins
+:author: dheerajodha
+:sig: gsoc
+description: >
+  The Plugin Health Scoring System is a Google Summer of Code 2022 project.
+opengraph:
+  image: /images/gsoc/2022/plugin-health-score/introducing-the-plugin-health-scoring-system.png
+links:
+  gitter: https://gitter.im/jenkinsci/GSoC-Plugin_Health_Score
+---
+
+image:/images/gsoc/2022/plugin-health-score/introducing-the-plugin-health-scoring-system.png[GSoC, height=420, role=center, float=center]
+
+The goal of this blog is to showcase the work done during the Google Summer of Code 2022 coding phases.
+
+Please see the link:/projects/gsoc/2022/projects/plugin-health-scoring-system/[project page] for a detailed description of the project.
+
+== About Project
+
+Plugin maintenance is an extremely important phase of a plugin’s lifecycle. Jenkins has always been active in ongoing developments which raises a need for thousands of plugins to catch up with their latest developments, in order to stay useful for a broader set of users.
+
+We aim to fulfil this need by introducing a metric system to calculate the health score of Jenkins plugins. This health score measures the plugin’s development maturity level with the help of accurate numeric figures. This score attempts to provide an accurate picture of how much care and help a plugin needs and makes it easier for contributors to contribute. This score also allows users to make a conscious decision before installing and/or using a plugin.
+
+There were 2 coding phases in the GSoC 2022.
+
+== Phase 1:
+
+In this phase, we worked on setting up the project and followed best practices while doing so. The whole application is dockerized so that it is easier for anyone to spin-up this project and test it out locally. There's continuous integration (CI) in-place for the project to make sure that new changes are tested automatically. The useful details from the Jenkins Update Center is stored inside the database to run analysis.
+
+=== Deliverables
+
+* Create a spring boot project connected with a PostgreSQL DB (link:https://github.com/jenkins-infra/plugin-health-scoring/pull/3[PR])
+* Dockerize the whole application (link:https://github.com/jenkins-infra/plugin-health-scoring/pull/7[PR])
+* Setup continuous integration for the project (link:https://github.com/jenkins-infra/plugin-health-scoring/pull/10[PR])
+* Store plugin details from the update center, inside the DB in JSON format (link:https://github.com/jenkins-infra/plugin-health-scoring/pull/18[PR])
+
+=== Resources
+
+* link:https://github.com/jenkins-infra/plugin-health-scoring/milestone/1?closed=1[View Milestone 1]
+* Jenkins Online Meetup Phase 1: link:https://docs.google.com/presentation/d/1t2vuNn1NFpDusnw0m4vdFw6WBQMeU6kccv_K1v2L6R0/edit#slide=id.g13dcaed2105_0_0[Slides]
+* Jenkins Online Meetup Phase 1: Demo and Presentation
+
+video::loLSNdCv6K4?t=1089[youtube,width=800,height=420,start=3219]
```

Comment:
The start time mentioned in the brackets seems to conflict with the intended point in the video. Also, mentioning the timestamp in the link is not necessary.
```suggestion
video::loLSNdCv6K4[youtube,width=800,height=420,start=1089]
```